src/main/java/com/contrastsecurity/sdk/ContrastSDK.java

```diff
@@ -100,6 +102,29 @@ public ContrastSDK(String user, String serviceKey, String apiKey, String restApi
         this.gson = new Gson();
     }
 
+    /**
+     * Create a ContrastSDK object that will attempt to use the Contrast V3 API

```

Comment:
Change this to 

`Create a ContrastSDK object to use the Contrast V3 API through a Proxy.`

Also change line 139 to

`Create a ContrastSDK object to use the Contrast V3 API`

Reply:
Done